test/wasi/test-wasi.js

```diff
@@ -26,3 +26,4 @@ testWasiPreview1(['preopen_populates']);
 testWasiPreview1(['stat']);
 testWasiPreview1(['sock']);
 testWasiPreview1(['write_file']);
+testWasiPreview1(['--target=wasm32-wasip1-threads', 'pthread']);
```

Comment:
It seems this has been flaking the CI for two month. https://github.com/nodejs/reliability/blob/main/reports/2025-07-19.md